src/main/java/org/taktik/icure/dao/impl/ClassificationTemplateDAOImpl.java

```diff
@@ -36,7 +38,7 @@
  * Created by dlm on 16-07-18
  */
 @Repository("classificationTemplateDAO")
-@View(name = "all", map = "function(doc) { if (doc.java_type == 'org.taktik.icure.entities.ClassificationTemplate' && !doc.deleted) emit( doc.patientId, doc._id )}")
+@View(name = "all", map = "function(doc) { if (doc.java_type == 'org.taktik.icure.entities.ClassificationTemplate' && !doc.deleted) emit( doc.label, doc._id )}")
```

Comment:
@cebamps can you have a look a this?
Is it compatible with `icc-api` master and our usage of it?

Reply:
@antoinepairet No problem with that change: we do not use the `all` view.

I guess the intent here is to change the view key to something that should be present in all templates. `doc.patientId` is always null because it is not a ClassificationTemplate field.